stor/s3.py

```diff
@@ -765,3 +769,34 @@ def to_url(self):
         """Returns HTTP url for object (virtual host-style)"""
         return u'https://{bucket}.s3.amazonaws.com/{key}'.format(bucket=self.bucket,
                                                                  key=self.resource)
+
+    def restore(self, tier='Bulk', days=10):
+        """Issue a restore command for a single object from glacier.
+
+        Args:
+            tier (str, optional): restore speed (see Glacier docs for details)
+            days (int, optional): number of days to keep data in S3 post-restore.
+        Returns:
+            dict or None: restore response from S3 client
+        Note:
+            Calling ``restore()`` on a directory will not work correctly. Only use this for single
+            objects! (you can always do ``[s.restore() for s in stor.list(<directory>)]``)
+
+        Ignores RestoreAlreadyInProgressError and AlreadyRestoredError (note that you can't force
+        S3 to do a faster restore once you've chosen a tier)
+        """
+        valid_tiers = ('Standard', 'Bulk', 'Expedited')
```

Comment:
could add constants instead, up to you

Reply:
I didn't because I didn't want to add more items to the API... I'm open to a PR if we want to do that, just wasn't sure where you'd stick them...